x-pack/legacy/plugins/siem/public/mock/global_state.ts

```diff
@@ -85,6 +85,11 @@ export const mockGlobalState: State = {
           dnsSortField: { field: NetworkDnsFields.queryCount, direction: Direction.desc },
           isPtrIncluded: false,
         },
+        tls: {
```

Comment:
Please add this to the `networkModel.NetworkTableType` and index by that key like the others